### ISSUE TITLE
custom menu/editor framelimit

### ISSUE THREAD

**Issue by Karoo13** [ppy/osu-stable-issues#292]
Since that update which changed the framelimit options to "optimal" and "power saving", power saving is forced outside of gameplay, which is too high for my computer. I would propose that menu and editor custom framelimits be added (config only), since these are quite different scenarios, and the current (lack of) options likely don't meet every user's preference.

About my own setup, I am on an optimus laptop with a second 144hz monitor attached. I have a custom framelimit of 251 set via config file (lowest multiple of 1ms where I get stable framerate, +buffer), and it is actually being increased to 288 when in the menu or editor. 
In menu, it stays 286+, though 145 would be enough. In editor, I average 280 with the power saving framelimit, while I would prefer it to be stable and not max out (same as gameplay, 251). The "optimal" setting is giving me 960, which is totally useless to me.

**Comment by peppy** [comment]
Can you use vsync?

**Comment by Karoo13** [comment]
Yes, vsync does keep 144fps in menu and editor. I tried both just now and vsync feels like it has slightly higher latency but is smoother (though I wouldn't trust my perception). 

Still, for players who use a regular framelimit it doesn't seem nice to force 2x monitor rate on them outside of gameplay, when that might be 120 (lower than non-gaming mouse polling rate) up to 480 (not exactly "power saving").
